packages/web-components/src/components/va-loading-indicator/va-loading-indicator.css

```diff
@@ -23,6 +23,19 @@
   animation: spin 1.5s linear infinite;
 }
 
+.loading-indicator.loading-indicator-inline {
```

Comment:
The parent class name (`'.loading-indicator`) can be removed for these classes, I think `.loading-indicator-inline` is specific enough to this component.